packages/components/src/Chips/InternalChipDismissible/InternalChipDismissibleInput.tsx

```diff
@@ -44,14 +47,13 @@ export function InternalChipDismissibleInput(props: ChipDismissibleInputProps) {
     handleDebouncedSearch,
   } = useInternalChipDismissibleInput(props);
 
-  const menuRef = useScrollToActive(activeIndex);
+  const scrollableRef = useScrollToActive(activeIndex);
   const { ref: visibleChildRef, isInView } = useInView<HTMLDivElement>();
 
   const {
-    styles: popperStyles,
-    attributes,
```

Comment:
worth double checking that `attributes` was nothing previously. I'm pretty sure it was.